tests/python/test_reduction.py

```diff
@@ -17,88 +26,89 @@ def fill():
         for i in a:
             a[i] = i
 
+    if op == OP_ADD:
```

Comment:
nit: Could you also create a look up table from `OP_X` to the Taichi/NumPy function?

Reply:
yep sure.